layouts/contact/list.html

```diff
@@ -3,55 +3,159 @@
 {{ .Content }}
 
 <div class="container">
-  <form action="https://forms.dc.scilifelab.se/api/v1/form/SBHJi6wO1oUz7ulb/incoming" class="needs-validation"
-    novalidate method="POST" accept-charset="utf-8">
-    <div class="row mb-3 mt-2">
-      <label for="name" class="col-sm-2 col-form-label fw-bold">Your name*:</label>
-      <div class="col-sm-10">
-        <input type="text" class="form-control" id="name" name="name" required>
+  <!-- Success message (shown after successful submission) -->
+  <div id="success-message" class="alert alert-success d-none" role="alert">
+    <h4 class="alert-heading">Thank you!</h4>
+    <p id="success-message-text">Your message has been sent successfully. We'll get back to you as soon as possible.</p>
+  </div>
+
+  <!-- Error message (shown if submission fails) -->
+  <div id="error-message" class="alert alert-danger d-none" role="alert">
+    <h4 class="alert-heading">Submission failed</h4>
+    <p id="error-message-text">There was an issue with your submission. Please try again.</p>
+  </div>
+
+  <!-- Contact form container -->
+  <div id="contact-form-container">
+    <form id="contact-form" class="needs-validation" novalidate>
+      <div class="row mb-3 mt-2">
+        <label for="name" class="col-sm-2 col-form-label fw-bold">Your name*:</label>
```

Comment:
I did not like _"Your name"_ from before, maybe we can take this opportunity to change this to just _"Name"_ ?